ui/components/HelmReleaseDetail.tsx

```diff
@@ -47,6 +48,9 @@ function helmChartLink(helmRelease: HelmRelease) {
 }
 
 function HelmReleaseDetail({ helmRelease, className, customTabs }: Props) {
+  const { data } = useFeatureFlags();
```

Comment:
:thinking: This strikes me as a bit odd. Since it appears `data` can be `undefined` here (probably while we are loading), it means things will pop in after the feature flag is loaded.

Not a blocker, but something to think about.